### DISCUSSION TITLE
NFT-DAO Reporting to Project Catalyst

### DISCUSSION THREAD

**Discussion by stephen-rowan** [NFT-DAO/Governance-HOLON#104] [General]
To Project Catalyst Moderator Team

In accordance with Project Catalyst’s Community Moderation Guidelines, I am “leading by action” and am writing with reference to NFT-DAO’s reporting to Project Catalyst. I submit that the reporting of the Fund 3 proposal Comprehensive NFT Framework Collab has been inaccurate, and misleading to the Project Catalyst community (https://cardano.ideascale.com/a/dtd/Comprehensive-NFT-Framework-Collab/334521-48088#idea-tab-comments). 

Since April 2021, the following events have not been reflected in NFT-DAO’s public reports:

1)	A unilateral decision was made by the original proposers of the Comprehensive NFT Framework Collab to join OccamFi’s “Hackathon”, without consulting NFT-DAO’s core contributors. This decision was presented as a fait accompli to NFT-DAO’s Governance group. The “Hackathon” was actually a submission to a funding vetting process.
2)	The entire core Development team left in May 2021, and this fact has not yet been fully communicated to NFT-DAO members. Nor have the reasons for their departure been documented in Project Catalyst reporting.
3)	Concerns which have been raised in meetings about the proposed partnership with OccamFi were initially treated by NFT-DAO proposers in a way that felt dismissive. These concerns focused on the fact that OccamFi, in its consultancy with NFT-DAO for funding, would demand proprietary working methods from NFT-DAO, which would mean the Comprehensive NFT Framework Collab is not an “open-source composable NFT framework” as stated in the proposal.
4)	The above events have led to two members of NFT-DAO’s Governance group ceasing any further involvement with NFT-DAO.
5)	Details about OccamFi’s deal with NFT-DAO (for example its valuation of NFT-DAO, and the amount of funding it is offering in return for what percentage of NFT-DAO) have not been communicated to members or core contributors.
6)	In preparation for a OccamFi AMA (Ask Me Anything), several questions were collated from members of NFT-DAO. Members’ trust in this AMA was then undermined because the host was not neutral, stated his support for the deal, and characterised the questions as “overly aggressive”.

A key factor in causing this situation may be that NFT-DAO has not yet agreed a charter/constitution, a group decision-making process, a code of conduct or a contributions policy. Members do “assume good intent” as per moderation guidelines; but without these things in place, decentralized governance is more difficult to achieve.

The members (the “implementers”) who have left NFT-DAO have contributed substantial work that is unrewarded (“Recognize and credit where it is due”), and have left, as a group, for reasons that should be captured by the Catalyst reporting process.
I fully support Project Catalyst’s experimental approach, but an experiment is only useful if it fully represents what has happened. The events detailed here are not adequately represented in NFT-DAO’s reports to Project Catalyst.

It would be appreciated if you could respond to these concerns before the next Catalyst Townhall on 26th May 2021, so that members who have left, and those who are yet to contribute, may be assured of full transparency in respect of the Comprehensive NFT Framework Collab proposal.

Yours sincerely
 Stephen Whitenstall

I have been part of NFT-DAO Governance since the beginning of March 2021, built up the project’s documentation, maintained its GitHub repositories, conducted governance token research & development and trained/mentored members. I helped draft the “NFT-DAO EZ-Honor contribution token proposal” with Troy Egan and Dan Verowski and am a co-proposer and GitHub repo maintainer for the “NFTDAO: Industry Standards 1 proposal”. I also feature in two NFT-DAO Youtube videos on “Governance & Contribution Token” and “Standards”. I have my own Fund 5 proposal “QA-DAO” and now contribute to two other Catalyst projects.

**Comment by tttroyyy** [comment]
RESPONSE
My name is Troy Egan. I am a co-proposer with the Project Catalyst Fund 3 NFT-DAO Framework Collab, I am lead of the governance team and a member of the NFT-DAO who are working toward deliverables for this project: https://cardano.ideascale.com/a/dtd/Comprehensive-NFT-Framework-Collab/334521-48088. 
This correspondence is in response to Stephen Whitenstall’s submission to the Project Catalyst Moderator Team that NFT-DAO’s reporting to the Project Catalyst community has been inaccurate and misleading. I write on behalf of the NFT-DAO core team.

Firstly, Stephen has been a valued member of NFT-DAO’s governance holon (team) with a range of contributions that he has outlined below. Stephen has chosen to leave the NFT-DAO after a meeting that was held on Thursday May 20 2021 7pm UTC, but has kindly opted to maintain the GitHub repositories for the NFT-DAO until a replacement is found. To clarify, Stephen is under no obligation from NFT-DAO to maintain these repositories, and no maintainer is currently being sought.  I am confident that the issues Stephen has raised below have contributed to his decision to leave the NFT-DAO. This response will firstly address the primary issue raised concerning NFT-DAO’s reporting, followed by addressing some of the issues themselves.

REPORTING CLAIMS
We disagree with Stephen’s claim of inaccurate and misleading reporting to Project Catalyst by NFT-DAO. The Project Catalyst cohort has a fortnightly reporting process, and two reports have been forwarded to date with a third due on the week of Monday May 24 2021, as per agreement with the cohort administrators. Stephen’s claim is negated on the following counts:

1. The last report was due and submitted on May 12 2021, prior to any of the events Stephen has claimed have not been, or needed to be, reported. These include any agreements with Occam.Fi (which were only at the discussion phase with one core team member) or any exit by developers;
2. Since Stephen’s claim was posted publicly in a Github repository on Sunday 9am UTC (approx.), this was also prior to the next cohort reporting period of Monday May 24 2021 UTC, which did not provide adequate time to report through the established channels.
3. Since this claim has not taken into account the fortnightly reporting periods in which NFT-DAO co-proposers are obligated to report to the cohort administrators, then Stephen’s claim of inaccurate reporting is itself inaccurate since the reporting cycle had not yet re-opened;
4. _If_ the events had occurred _within_ the reporting windows, the range of questions for the last report were general in nature, requesting quantifiable information such as numbers of users, transactions, launch dates and additional information. There was no requirement within these parameters to divulge the level of reporting that Stephen himself expects NFT-DAO to provide to the cohort. This is understandable since most projects are still in development and the reporting mechanism is sensitive to this; 
5. If we have in someway failed to report in the ways that the cohort administrators are happy with – as Stephen claims, then we are more than willing to discuss and comply with the appropriate measures. However, if Stephen is suggesting that the NFT-DAO and other cohort members be required to report according to a method of his personal preference, then that is between the Catalyst community and Stephen to discuss.

ISSUES RAISED

This next section will address individual issues raised by Stephen himself, which may have contributed to his departure from NFT-DAO:

_"1. A unilateral decision was made by the original proposers of the Comprehensive NFT Framework Collab to join OccamFi’s “Hackathon”, without consulting NFT-DAO’s core contributors. This decision was presented as a fait accompli to NFT-DAO’s Governance group. The “Hackathon” was actually a submission to a funding vetting process."_

This statement is inaccurate. A unilateral decision on this issue was NOT made by all original proposers. This is a blanket statement that includes one co-proposer who never participated in the DAO (Finley) and another (Troy) who opted for a consultation process: to draft and send a poll to gauge interest amongst core contributors prior to any formal commitments. This second point was reiterated in an open Governance Team meeting dated Thursday UTC 7pm. Discussion of the opportunity had occurred in recorded NFT-DAO townhalls (with some notes taken by Stephen), within the NFT-DAO Discord server, in a preliminary poll and in a future AMA with Occam.Fi (with some questions edited by Stephen). 

_"2. The entire core Development team left in May 2021, and this fact has not yet been fully communicated to NFT-DAO members. Nor have the reasons for their departure been documented in Project Catalyst reporting."__

This statement is not factual. On May 14 2011, two days after the last reporting window, three developers opted to depart the project once their portions of the project are complete. There are at least four more developers in the core dev team who are staying. The next Project Catalyst reporting window opens AFTER this claim was made and many of these points have been addressed there. (Note: the cohort report contains sections that are not available to the public. That will need to be addressed with the Project Catalyst team).

_"3. Concerns which have been raised in meetings about the proposed partnership with OccamFi were initially treated by NFT-DAO proposers in a way that felt dismissive. These concerns focused on the fact that OccamFi, in its consultancy with NFT-DAO for funding, would demand proprietary working methods from NFT-DAO, which would mean the Comprehensive NFT Framework Collab is not an “open-source composable NFT framework” as stated in the proposal."__

Stephen had expressed in conversation his key concern was that Occam.Fi was a conflict of interest with NFT-DAO as an open-source project. He was also not the only one to feel reservations as atleast one other person felt similarly. Both people had made it known that they would remove themselves from DAO activity until the Occam.Fi issue was clarified. When NFT-DAO received a surprising confirmation from Occam.Fi that it was a successful finalist in a hackathon that had never even taken place, both people publicly exited the community. As an NFT-DAO proposer, I did not intend in anyway that my contributions be interpreted as dismissive. However, it was made very clear that Stephen is committed to open-source projects to such a degree, and that Occam.Fi posed such a potential threat to him (and at least one other person), that they would leave NFT-DAO if it were to engage in any kind of relationship with Occam.Fi. From a personal values point of view this makes a lot of sense. For other members of NFT-DAO, Occam.Fi is still only a discussion point.

_"4. The above events have led to two members of NFT-DAO’s Governance group ceasing any further involvement with NFT-DAO."_

As addressed above.

_"5. Details about OccamFi’s deal with NFT-DAO (for example its valuation of NFT-DAO, and the amount of funding it is offering in return for what percentage of NFT-DAO) have not been communicated to members or core contributors."_

There is currently no “deal” or agreement with Occam.Fi so these details do not exist from NFT-DAO’s position. As for questions to Occam.Fi, the AMA and AMA questions that Stephen helped edit are still pending.

_"6. In preparation for a OccamFi AMA (Ask Me Anything), several questions were collated from members of NFT-DAO. Members’ trust in this AMA was then undermined because the host was not neutral, stated his support for the deal, and characterised the questions as “overly aggressive”."_

This is another blanket statement and is problematic as it gives the impression that ALL members have reacted to Occam.Fi in the same way as Stephen has. This is not the case. Five of the six points that have been raised as examples of NFT-DAO’s inaccurate or misleading reporting, deals with Occam.Fi as if they are an actual NFT-DAO partner, which they are not. It is agreed, the communication and decision-making required of NFT-DAO as an “emergent” collaborative community can be better, and we – like any emerging DAO – will always aim to iterate and improve. 

However, it can be argued that Stephen’s (and other’s) position with regard to Occam.Fi is not neutral and this resonates in the points he is forwarding. From my own position, I am still doing my own due diligence regarding Occam.Fi (and any other partner opportunity) and will be seeking consensus from core contributors regarding these matters. 

_"A key factor in causing this situation may be that NFT-DAO has not yet agreed a charter/constitution, a group decision-making process, a code of conduct or a contributions policy. Members do “assume good intent” as per moderation guidelines; but without these things in place, decentralized governance is more difficult to achieve."_

This is agreed. A code of conduct and contributions policy are the tasks that the governance team (including myself, and the two members who have exited) had been tasked with completing. These tasks have not been completed and will need to be resolved.

_"The members (the “implementers”) who have left NFT-DAO have contributed substantial work that is unrewarded (“Recognize and credit where it is due”), and have left, as a group, for reasons that should be captured by the Catalyst reporting process.
I fully support Project Catalyst’s experimental approach, but an experiment is only useful if it fully represents what has happened. The events detailed here are not adequately represented in NFT-DAO’s reports to Project Catalyst."_

Contributions and rewards to contributors of Fund3 deliverables have not ceased being a priority, and neither has a more robust tokenomics model that can sustain a collaborative economy in the future. As NFT-DAO is modelled after Project Catalyst’s experimental and iterative approach, we are grateful to past, present and future participants who are helping to develop and grow the processes and principles that can make this project thrive. My hope is that, as we “break things” through experimentation, people don’t leave bitter or resentful that their personal expectations weren’t met, but that the project became something more than any one individual or viewpoint.

_"It would be appreciated if you could respond to these concerns before the next Catalyst Townhall on 26th May 2021, so that members who have left, and those who are yet to contribute, may be assured of full transparency in respect of the Comprehensive NFT Framework Collab proposal.
Yours sincerely
Stephen Whitenstall"_

This response has been forwarded to The Project Catalyst Moderation Team on Ideascale

CONCLUSION
Stephen has proved himself to be an industrious and contentious contributor to the community with a strong ethos toward open-sourced initiatives and transparency. In conversation, Stephen had informed that this is what had drawn him to the NFT-DAO project. Unfortunately, this rigid approach to transparency and open-sourcing is not easily conducive to an emergent community or commercial enterprise; but it also highlights the challenges of building collaborative communities from scratch.

We agree with many of the criticisms and insights Stephen has raised as a means to make NFT-DAO better. As an “emergent” DAO, there has arisen some conflicting opinion about governance issues, tokenomic rewards, decision-making, degrees of transparency and open-sourcing. These issues have not been able to be resolved according to the expectations of some members. Although we are not happy to see him leave, we support Stephen’s recent decision to depart from NFT-DAO as we honor personal free will, independence and self-sovereignty. 

However, we find Stephen’s claims that NFT-DAO has been reporting inaccuracies and misleading information to Project Catalyst to be surprising, for the reasons noted above. Since Stephen has positioned himself in the Project Catalyst community as the proposer of a QA-DAO (overseeing Quality Assurance) – a service both welcomed and needed in the space – it is concerned that Stephen’s assessment of NFT-DAO’s reporting appears to have been so speculative, biased and inaccurate in its own right. At best, it has provided Stephen, ourselves and hopefully others an opportunity to clarify our communications with the cohort administrators and community about Catalyst reporting. At worst, it creates a speculative platform to fuel further dissent for those whose expectations have not been met, which is not the domain of NFT-DAO.

_"I have been part of NFT-DAO Governance since the beginning of March 2021, built up the project’s documentation, maintained its GitHub repositories, conducted governance token research & development and trained/mentored members. I helped draft the “NFT-DAO EZ-Honor contribution token proposal” with Troy Egan and Dan Verowski and am a co-proposer and GitHub repo maintainer for the “NFTDAO: Industry Standards 1 proposal”. I also feature in two NFT-DAO Youtube videos on “Governance & Contribution Token” and “Standards”. I have my own Fund 5 proposal “QA-DAO” and now contribute to two other Catalyst projects."_

For the sake of honesty and transparency, both Stephen Whitenstall and Dan Verowski have made the following public exit statements regarding their participation in NFT-DAO and NFT-DAO governance (viewable at: https://cardano.ideascale.com/a/dtd/NFT-DAO-EZ-Honor-contribution-token/352773-48088#idea-tab-comments)

Dan Verowski (@dan.mercurius) 
Team Member
“For the sake of honesty and transparency I hereby state that from today onwards I'm no longer actively working within NFTDAO or the NFTDAO governance Holon. 
My workload will be distributed among the remaining governance Holon members and I'm positive that they will excel. 
Stay mindful, healthy & happy”

Stephen Whitenstall (@swhitenstall) 
Team Member
“For the sake of honesty and transparency I also state that I am no longer actively working within NFTDAO or any other NFTDAO governance work. Some context for this can found in the NFT-DAO Governance Discussion Boards.
I continue to be the maintainer of the Governance-HOLON, NFT-Standards and NFT-DAO GitBook repositories until these can be handed over.”

**Reply by DanM3rcurius** [reply]
Dear @tttroyyy You got to build sth first in order to break it.

I will have a more expanded statement on what has happened and why I left published soon. But here's a short reply to your answer to @stephen-rowan 

Interesting to see how *"Contributions and rewards to contributors of Fund3 deliverables have not ceased being a priority"* and the [most active members](https://github.com/NFT-DAO/Governance-HOLON/graphs/contributors) of the governance holon (that was working on contribution tracking and reward distribution) leaving fit together?

> " A code of conduct and contributions policy are the tasks that the governance team (including myself, and the two members who have exited) had been tasked with completing. These tasks have not been completed and will need to be resolved."

How can this be achieved by a single "holon" if all "holons" have to do this in concert, for it to have a consensus about the common goals, values and purpose? Stephen and me mentioned the pinnacle and primordial importance of this document to commence any kind of collective activity. We were told by the core team to just create it within the governance "holon" and then push it into the other "holons". All the while a clear goal was changing almost on a daily basis and buzzwords like chaotic and emerging where used to cover for shortcomings of initial documentation. 

The goal here is not to point fingers or harm reputations, but to learn from past mistakes and move forward in a more balanced direction.
I'm taking the lessons learnt and new friendships over to the projects I'm working in now and am thankful for huge learning experience within NFTDAO.

We have this incredible tool (DLT) in our hands now that we can deploy to create transparent supply chains, closed loop economies and true value societies that takes into account the balance of natural emerging systems (like our planetary ecosystem) and integrates them as a stakeholder in a capitalism that is nurturing not neutering. That can only be accomplished  if we start acting honest towards our Selfs and others, now.

Hopefully this will serve as a learning experience for all parties and we can grow and prosper next to each other. 

Stay mindful, healthy & happy :pray:

**Reply by tttroyyy** [reply]
Hi Dan

Thank you for your considerate reply. I appreciate the kind spirit in which it was delivered. Yes, there has been plenty of learning experience from the last five months. It's given me a whole chapters worth of content for my PhD.

If you and @stephen-rowan don't mind, I will be incorporating these points into my own assessment of the breakdown of the initial core contributor team, and will post accordingly.

Warm Regards
Troy

**Comment by DanM3rcurius** [comment]
What do you mean? I don't understand what you want to reference where?

cheers,
D.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Reply by tttroyyy** [reply]
Hey Dan. To clarify, I'll be referencing public statements about NFT-DAO (by past and former members) into my own public assessment of where the project went well and where it could be improved. I guess, since the statements are public, there isn't any need for permissions. Apologies for any confusion.

**Comment by DanM3rcurius** [comment]
Dear Troy,

im guessing there will be many more public Statement to come.
Not sure why you need to announce your assessment, rather than just share it in the respective forum?

It would be mindful to share assessments sooner than later, as the events are continuously unfolding, especially with the community affected.

Sincerely
D.

**Reply by tttroyyy** [reply]
Thank you Dan

Yes, I think there will be a lot of mileage out of what we can learn from our experiences in NFT-DAO. Hopefully the upside is, people will have a better understanding of what its like trying to build a collaborative project from scratch.

As per my assessment, I don't 'need' to announce it. I'm doing so in a spirit of open communication.

Thank you for your suggestion about mindful sharing. I believe mindfulness to be very important too. Personally, I'm pacing myself around this assessment for a few reasons: a) it is time consuming, but requires attention to try and capture everyone's viewpoints which are spread out across various platforms, b) Since I haven't received a cent from NFT-DAO, I'm writing this assessment in a voluntary capacity. Therefore, I'll need to balance this in relation to my other responsibilities (which I guess is mindfulness in practice); c) I am part of the community affected, and I have found the disappointments and challenges demotivating (And as you said, there may be many more public statements, so there is no end date to the discussion).

I'm also hopeful that some good things can come from people learning to work through challenges and finding common ground on the other side. If we can't, then there doesn't seem to be much hope in this overly divisive world. 

Cheers, Troy
